docs/classes/auth0client.html

```diff
@@ -2830,6 +2830,9 @@
 		<li class=" tsd-kind-variable">
 			<a href="../globals.html#cache_location_memory" class="tsd-kind-icon">CACHE_<wbr>LOCATION_<wbr>MEMORY</a>
 		</li>
+		<li class=" tsd-kind-variable">
+			<a href="../globals.html#invalid_refresh_token_error_message" class="tsd-kind-icon">INVALID_<wbr>REFRESH_<wbr>TOKEN_<wbr>ERROR_<wbr>MESSAGE</a>
+		</li>
```

Comment:
Is it expected to have these consts in the docs ?

Reply:
I guess they could be useful, but they're more for internal use. I would hide them from the docs.

Reply:
Removed them from the docs